backend/src/middleware/email/templates/passwordReset.js

```diff
@@ -1,83 +0,0 @@
-import CONFIG from '../../../config'
-
-export const resetPasswordMail = options => {
-  const {
-    name,
-    email,
-    nonce,
-    subject = 'Use this link to reset your password. The link is only valid for 24 hours.',
-    supportUrl = 'https://human-connection.org/en/contact/',
-  } = options
-  const actionUrl = new URL('/password-reset/change-password', CONFIG.CLIENT_URI)
-  actionUrl.searchParams.set('nonce', nonce)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-18T21:50:49Z" title="Wednesday, September 18th 2019, 11:50:49 pm +02:00">Sep 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -53,6 +54,7 @@ export const wrongAccountMail = options => {
   } = options
   const actionUrl = new URL('/password-reset/request', CONFIG.CLIENT_URI)
   return {
+    from: '"Human Connection" <info@human-connection.org>',
```


We already forgot to set this. Can we store this somewhere at a central location, so we never forget to add it? Seeing it twice in the same file makes me nervous.

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-18T21:54:10Z" title="Wednesday, September 18th 2019, 11:54:10 pm +02:00">Sep 18, 2019</time>_

---


Well, it feels more natural to have it there. Maybe it should be the task of QR to not have it slip through. :thinking:

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-09-18T22:26:14Z" title="Thursday, September 19th 2019, 12:26:14 am +02:00">Sep 19, 2019</time>_

---


I also felt it made more sense to place the `from` alongside the other attributes. But I'm hoping to refactor the passwordReset in such a way that I can get rid of the duplicate!